tests/unittest/disaggregated/test_coordinator_e2e.py

```
@@ -0,0 +1,430 @@
+            self._session, self._router.servers
+        )
+        if ready:
+            await self._router.prepare_servers(ready)
+        return ready, unready
+
+    async def shutdown(self):
+        await self._session.close()
+
+
+def _make_config(ctx_url, gen_url, public_port):
+    host_port = lambda u: (u.split(":")[0], int(u.split(":")[1]))
```

Ruff (0.15.20)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `host_port` as a `def`

(E731)